packages/stg-evm-usdt-v2/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "tether-contracts-hardhat",
```

Comment:
This should be `@stargatefinance/usdt-evm`

Reply:
Oo yes fixed! 10q